### ISSUE TITLE
Bias Term in DepthWiseXCorr

### ISSUE THREAD

**Issue by maxizi** [STVIR/pysot#313]
Hi,

I have a very special question as it really is just a minor detail, but somehow I think there must be a reason for this which I am currently not aware of. In the DepthwiseXCorr class (pysot/models/head/rpn.py) we have 2D conv layers with explicitly no bias. What is the reason for explicitly not using bias? The last conv layer again has a bias term.... I cannot think of any advantage and did not find an explanation elsewhere.

I would be very happy if someone has an idea and could share it :)

Thanks!
![image](https://user-images.githubusercontent.com/36971632/76216923-ba6ea180-6211-11ea-9879-b265e10e6c59.png)

**Comment by Grundwissen** [comment]
The bias can be ignored since its effect will be canceled by the subsequent mean subtraction of the batch normalization.
Take a look at the batch normalization paper: [https://arxiv.org/pdf/1502.03167.pdf](link)
Or at PyTorch forum: [https://discuss.pytorch.org/t/any-purpose-to-set-bias-false-in-densenet-torchvision/22067](link)

**Comment by maxizi** [comment]
Thanks a for the quick reply and for pointing me to the right resources :)

This makes sense, issue solved.
